generators/server/templates/src/test/java/package/CassandraKeyspaceIT.java.ejs

```diff
@@ -23,9 +23,15 @@ import com.datastax.driver.core.Session;
 import org.junit.jupiter.api.Test;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.boot.test.context.SpringBootTest;
+<%_ if (messageBroker === 'kafka') { _%>
+import org.springframework.kafka.test.context.EmbeddedKafka;
+<%_ } _%>
 
 import static org.assertj.core.api.Assertions.assertThat;
 
+<%_ if (messageBroker === 'kafka') { _%>
+@EmbeddedKafka
```

Comment:
Is it needed to have an EmbeddedKafka in all tests ?

Reply:
Thanks a lot for starting to review my PR @cbornet 

I pushed quickly what I did before vacations, so not really sure about this part.
And I don't like this part, as I added `@EmbeddedKafka` everywhere. I started to use a dedicated `@Configuration` in `EmbeddedKafkaConfiguration` but I needed to customize the Embedded Kafka myself and didn't manage to make it works.

So ideas are welcome here